pyproject.toml

```diff
@@ -12,6 +12,7 @@ include = ["cognite_toolkit/*", "cognite_toolkit/**/*"]
 python = ">=3.9,<3.12"
 python-dotenv = "^1.0.0"
 cognite-sdk = {version = ">=7.17.1, <7.18", extras = ["pandas"]}
+cognite-logger = "^0.6"
```

Comment:
I think this should be a dev dependency and not required for everyone that install the toolkit.

Reply:
I have moved it.

Reply:
agree